neurom/fst/_neuritefunc.py

```diff
@@ -182,6 +182,13 @@ def total_length_per_neurite(neurites, neurite_type=NeuriteType.all):
                 for n in iter_neurites(neurites, filt=is_type(neurite_type)))
 
 
+def terminal_length_per_neurite(neurites, neurite_type=NeuriteType.all):
```

Comment:
I would modify the name to "terminal_path_length" to avoid possible confusion between length and path_length.